### PR TITLE
chore: for simple test cases use createDefaultServer

### PR DESCRIPTION
**Pull request by hugoArregui**
Create a default database (one per runner) and create a default server pointing to such database once per test suite.

### CODE REVIEW

File: content/src/ports/activeEntities.ts [hunk 1]
```diff
@@ -74,6 +79,7 @@ export function createActiveEntitiesComponent(
 
   const collectionUrnsByPrefixCache = new LRU<string, string[]>({
     ttl: 1000 * 60 * 60 * 24, // 24 hours
+    max: components.env.getConfig(EnvironmentConfig.ENTITIES_CACHE_SIZE), //TODO
```
Comment (hugoArregui) [inline content/src/ports/activeEntities.ts:82]: Not quite sure what the proper limit is

Comment (hugoArregui) [reply content/src/ports/activeEntities.ts:82]: @pedrotambo does this look ok? should I use a different value?

**Review by marianogoldman — APPROVED** [review]

File: content/test/integration/controller/active-entity-by-content-hash.spec.ts [hunk 2]
```diff
@@ -1,21 +1,31 @@
...
-      .withConfig(EnvironmentConfig.DISABLE_SYNCHRONIZATION, true)
-      .andBuild()
+  beforeAll(async () => {
+    server = await createDefaultServer()
+    makeNoopValidator(server.components)
+  })
 
-    await server.startProgram()
+  beforeEach(async () => {
+    resetServer(server)
```
Comment (aleortega) [inline content/test/integration/controller/active-entity-by-content-hash.spec.ts:17]: missing await?

File: content/test/integration/controller/active-pointers.spec.ts [hunk 3]
```diff
@@ -19,29 +19,31 @@ interface DeploymentsRow {
...
+    server = await createDefaultServer()
     makeNoopValidator(server.components)
     makeNoopServerValidator(server.components)
-    await server.startProgram()
   })
 
-  afterEach(async () => {
-    await server.stopProgram()
+  beforeEach(async () => {
+    resetServer(server)
```
Comment (aleortega) [inline content/test/integration/controller/active-pointers.spec.ts:36]: same here 👼

File: content/test/integration/controller/deployment-pagination.spec.ts [hunk 4]
```diff
@@ -44,6 +39,16 @@ describe('Integration - Deployment Pagination', () => {
     E3 = await buildDeployData([P3], { type, timestamp: timestamp + 1, metadata: { a: 'metadata3' } })
   })
 
+  beforeEach(() => resetServer(server))
```
Comment (aleortega) [inline content/test/integration/controller/deployment-pagination.spec.ts:42]: and here

File: content/test/integration/controller/entity-metadata.spec.ts [hunk 5]
```diff
@@ -1,22 +1,29 @@
...
-      .andBuild()
+  let server: TestProgram
 
+  beforeAll(async () => {
+    server = await createDefaultServer()
     makeNoopValidator(server.components)
+  })
 
-    await server.startProgram()
+  beforeEach(() => resetServer(server))
```
Comment (aleortega) [inline content/test/integration/controller/entity-metadata.spec.ts:16]: and here, I hate adding these kind of repetitive comments, I'm sorry haha but maybe they are useful for you to identifying the spots faster.

File: content/test/integration/controller/deployment-pagination.spec.ts (hunk 4 above)
Comment (hugoArregui) [reply content/test/integration/controller/deployment-pagination.spec.ts:42]: I think this is ok, it returns a promise, so it will be awaited

**Review by aleortega — APPROVED** [review]

**Pull request merged**